environs/bootstrap/prepare.go

```diff
@@ -235,6 +235,7 @@ func prepare(
 	details.BootstrapConfig.Cloud = args.Cloud.Name
 	details.BootstrapConfig.CloudRegion = args.Cloud.Region
 	details.BootstrapConfig.CloudCACertificates = args.Cloud.CACertificates
+	details.BootstrapConfig.SkipTLSVerify = args.Cloud.SkipTLSVerify
```

Comment:
Log a warning if SkipTLSVerify is true. Also if CACeritifcates is provided as well, error.